docs_user/assemblies/assembly_adopting-the-data-plane.adoc

```diff
@@ -6,12 +6,13 @@
 
 Adopting the {rhos_long} data plane involves the following steps:
 
-. Stopping any remaining services on the {rhos_prev_long} ({OpenStackShort}) control plane.
-. Deploying the required custom resources.
-. If applicable, performing a fast-forward upgrade on Compute services from {OpenStackShort} {rhos_prev_ver} to {rhos_acro} {rhos_curr_ver}.
+. Stop any remaining services on the {rhos_prev_long} ({OpenStackShort}) {rhos_prev_ver} control plane.
+. Deploy the required custom resources.
+. If applicable, perform a fast-forward upgrade on Compute services from {OpenStackShort} {rhos_prev_ver} to {rhos_acro} {rhos_curr_ver}.
+. If applicable, adopt Networker nodes to the {rhos_acro} data plane.
 
 [WARNING]
-After the {rhos_acro} control plane is managing the newly deployed data plane, you must not re-enable services on the {OpenStackShort} {rhos_prev_ver} control plane and data plane. Re-enabling services causes workloads to be managed by two control planes or two data planes, resulting in data corruption, loss of control of existing workloads, inability to start new workloads, or other issues.
+Once the {rhos_acro} control plane is managing the newly deployed data plane, you must not re-enable services on the {OpenStackShort} {rhos_prev_ver} control plane and data plane. Re-enabling services causes workloads to be managed by two control planes or two data planes, resulting in data corruption, loss of control of existing workloads, inability to start new workloads, or other issues.
```

Comment:
The second part of the first sentence translates to "...you must not re-enable services on the RHOSP 17.1 control plane and data plane." Is "RHOSP 17.1" correct?